### PR TITLE
Tools provisioning

### PR DESCRIPTION
**Pull request by mbiarnes**
**Thank you for submitting this pull request**

Added missing DSLs for all jobs in Tools/Provisioning folder

<pre>
How to retest a PR or trigger a specific build:

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
</pre>

### CODE REVIEW

File: job-dsls/jobs/tools_provisioning_verdaccio.groovy [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+            }
+        }
+
+        // Allows a job to check out sources from an SCM provider.
+        scm {
+            git {
+                remote {
+                    url('git@github.com:kiegroup/kie-jenkins-scripts.git')
+                    credentials("kie-qe-ci-user-key")
+                    branch('*/BXMSPROD-1147')
```
Comment (almope) [inline job-dsls/jobs/tools_provisioning_verdaccio.groovy:69]: @mbiarnes is this branch correct?

Comment (mbiarnes) [reply job-dsls/jobs/tools_provisioning_verdaccio.groovy:69]: @Ginxo ^^

Comment (mareknovotny) [reply job-dsls/jobs/tools_provisioning_verdaccio.groovy:69]: maybe we can use standard branch or a tag?

Comment (Ginxo) [reply job-dsls/jobs/tools_provisioning_verdaccio.groovy:69]: it should be master

**Pull request merged**